src/util/fpclassify.cpp

```diff
@@ -1,8 +1,12 @@
-// this is a wapper around the fpclassify functions which prevents inlining 
-// It is compiled without optimization 
-// The rest of the source of Mixxx is compiled with -ffast-math, which breaks 
+// this is a wapper around the fpclassify functions which prevents inlining
```

Comment:
```suggestion
// this is a wrapper around the fpclassify functions which prevents inlining
```
this just jumped into my eye..